build/optaplanner-build-parent/pom.xml

```diff
@@ -267,7 +267,7 @@
       <dependency>
         <groupId>org.jboss.narayana.jta</groupId>
         <artifactId>narayana-jta</artifactId>
-        <version>5.11.2.Final</version>
+        <version>5.12.0.Final</version>
       </dependency>
```

Comment:
This is a leftover from https://github.com/kiegroup/optaplanner/pull/1464. In fact, `narayana-jta` is defined in the `quarkus-bom` too and we don't need to define it ourselves.

@radtriste Would you please remove this dependency entirely so that we avoid merge conflicts?

Reply:
done